demisto_sdk/commands/init/initiator.py

```diff
@@ -855,6 +1241,46 @@ def fix_test_file_import(self, name_to_change: str):
         ) as fp:
             fp.write(file_contents)
 
+    def get_suffix_xsiam_content(
```

Comment:
this shouldn't be part of the object, it doesn't use anything from `self`, please move out of the object.